src/components/Search.js

```diff
@@ -0,0 +1,26 @@
+import '../css/Search.css'
+import React, { useState } from 'react'
+import { useSearchParams } from 'react-router-dom'
+
+const Search = ({ searchedMovies, searchMovies }) => {
+  const [searchState, setSearchState] = useState(searchedMovies)
+  let [searchParams, setSearchParams] = useSearchParams()
+
+  const handleChange = (e) => {
+    setSearchParams({ search: e.target.value })
+    setSearchState(searchParams)
+    searchMovies(searchParams)
+  }
+
+  return (
+      <input
+        className="text-input"
+        type="text"
+        placeholder="search..."
+        value={searchState}
+        onChange={e => handleChange(e)}
+      />
+  )
+}
+
+export default Search
```

Comment:
Just wanted to make a quick note that I specifically used hooks for this component because I really wanted to practice using them on this project.  Last 'Rancid Tomatillos' I changed almost everything to hooks on Sunday, but I promise not to do that now.